### PR TITLE
Add VSTS ServerRequest entity

### PR DESCRIPTION
**Pull request by cristinamanum**
Add VSTS ServerRequest entity

### CODE REVIEW

File: src/NuGet.Services.Validation/Entities/SymbolsServerRequest.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        /// <summary>
+        /// The key of the symbols package.
+        /// </summary>
+        public int SymbolsKey { get; set; }
+
+        /// <summary>
+        /// The request name used for ingestion.
+        /// </summary>
+        public string RequestName { get; set; }
```
Comment (chenriksson) [inline src/NuGet.Services.Validation/Entities/SymbolsServerRequest.cs]: We provide the VSTS name? How will we set it?

File: src/NuGet.Services.Validation/Migrations/201807252343358_AddSymbolsServerRequest.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    public partial class AddSymbolsServerRequest : DbMigration
+    {
+        public override void Up()
+        {
+            CreateTable(
+                "dbo.SymbolsServerRequests",
+                c => new
+                    {
+                        SymbolsKey = c.Int(nullable: false, identity: true),
+                        RequestName = c.String(),
```
Comment (chenriksson) [inline src/NuGet.Services.Validation/Migrations/201807252343358_AddSymbolsServerRequest.cs]: Should this be `nullable: false`?

File: src/NuGet.Services.Validation/Entities/SymbolsServerRequest.cs (hunk 1 above)
Comment (cristinamanum) [reply src/NuGet.Services.Validation/Entities/SymbolsServerRequest.cs]: The ingestor job will set it.

File: src/NuGet.Services.Validation/Migrations/201807252343358_AddSymbolsServerRequest.cs (hunk 2 above)
Comment (cristinamanum) [reply src/NuGet.Services.Validation/Migrations/201807252343358_AddSymbolsServerRequest.cs]: Yes - correct . Updated

**Pull request merged**